stable/pachyderm/templates/pachd_rolebinding.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 {{- if .Values.rbac.create }}
 apiVersion: rbac.authorization.k8s.io/v1beta1
-kind: RoleBinding
+kind: ClusterRoleBinding
```

Comment:
This change should make the pipeline parameter `coefficient` work again.

Reply:
Can you please test this?

Reply:
Is there an issue for this I can look at? This is not an issue I ran into.